tests/GENERIC/equivalence_constant_propagation.sgrep

```diff
@@ -0,0 +1 @@
+password("...")
```

Comment:
would this be better to be"$F("password")" to fully test that we are propagating const Bar's value to password function so that we are matching it there?

Reply:
Also, I am not clear what "..." does compared to ..., can you explain?

Reply:
yes I may have chosen a bad name. 
It's supposed to emulate the boto check where we check that we don't pass a constant string as the password_account keyword parameter. The goal here is just to check that we pass a constant string to a specific function and that we handle the propagation of constant.

Reply:
"..." is matching any string (a sequence of any character), ... is matching a sequence of arguments (in a call context).

Reply:
We can match on that constant right? Like $F("password") would match the example.

Reply:
And "..." is a special case of ... where it matches on const literal strings. Does it match on const number?

Reply:
there are quotes around it, so it's a string, and it's just for strings.

Reply:
Yes, $F("password") should match!

Reply:
it would actually be a better test indeed, less confusing than using "..."

Reply:
Thanks for clarifying!